surface/include/pcl/surface/marching_cubes.h

```diff
@@ -380,6 +380,8 @@ namespace pcl
         percentage_extend_grid_ (percentage_extend_grid),
         iso_level_ (iso_level) 
       {
+        /** \brief Set grid resolution */ 
+        res_x_ = 32; res_y_ = 32; res_z_ = 32;
```

Comment:
Is class member default initialization a c++17 feature? We can also use the initializer list here

Reply:
No, it's in c++14, so we can use it.

Reply:
According to [C.45](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-default), we should use default initializer instead of initializer list, for this case.
Should I proceed in that direction?

Btw, thanks for pointing that out!

Reply:
I don't think we have explicitly adopted CppCoreGuidelines, but generally, I see no reason not to follow their suggestions. So yes, please go ahead.

Reply:
Please use default initializer since that is already used in several places in PCL for data members.